src/main/java/graphql/schema/DataFetchingEnvironment.java

```diff
@@ -270,4 +272,15 @@ public interface DataFetchingEnvironment extends IntrospectionDataFetchingEnviro
      * @return the coerced variables that have been passed to the query that is being executed
      */
     Map<String, Object> getVariables();
+
+
+    /**
+     * A method that should only be used by the GraphQL Java library itself.
+     * It is not intended for public use.
+     *
+     * @return an internal representation of the DataFetchingEnvironment
+     */
+    @Internal
+    Object toInternal();
```

Comment:
make it defaulted - so other peoples tests dont fail.  Thye might have mocked this interface and now this is a breaking change

Reply:
BREAKING CHANGE!!

Reply:
done